lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -116,9 +116,7 @@ define([
         showErrorMessage: function (message) {
             this.$('.wrapper-msg')
                 .removeClass('is-hidden')
-                .find('.msg-content .copy').text(message)
-                .closest('.msg')
-                .focus();
```

Comment:
By deleting `focus()`, we don't want start reading this error message by screen readers when it is appear. Am I right?